test/spec/features/service/link_spec.rb

```diff
@@ -1,44 +1,43 @@
 describe 'service link' do
   after(:each) do
-    %w(test-1 test-2).each do |s|
-      run "kontena service rm --force #{s}"
-    end
-    run "kontena stack rm --force simple"
+    run "kontena service unlink simple/bar test-1"
```

Comment:
Ideally scope these cleanups into their own context, because they're not relevant for all of the examples.

```ruby
context 'with a linked stack service` do
  before do
    with_fixture_dir("stack/links") do
      run! 'kontena stack install --no-deploy links.yml'
    end
  end
  after do
    run 'kontena service unlink simple/bar test-1'
    run 'kontena stack rm --force simple'
  end
  ...
end
```

Reply:
Feel free to fix if fingers are itching, I'm out for the weekend. 👍